community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -1834,7 +1834,7 @@
         "%%writefile ./custom_container/Dockerfile\n",
```

Comment:
<p>There are alternative prediction formats besides JSON? Is what you're saying here that JSON is the only one applicable for images?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='231'/>

Reply:
<p>We are using [Torchserve's KServe based inference API](https://pytorch.org/serve/inference_api.html#kserve-inference-api) which is also Vertex AI predictions compatible envelope and it's in JSON format. </p>

Reply:
Consider elaborating a bit more on your comment then, and linking to that spec (if you haven't already in the notebook)?

Reply:
Added reference link to the spec and Vertex AI Prediction input format for custom containers